src/database/bank_database.rb

```diff
@@ -0,0 +1,46 @@
+# a class created to simulate a bank database with user accounts, bank accounts and an ATM unit
+class BankDatabase
+
+    # constructor
+    def initialize(users, bank_accounts, transactions, atm_unit)
+        @users = users
```

Comment:
Are you using two or four spaces for indentation? In ruby it's recommended to use two spaces

Reply:
I used tab when I wanted to indent a new line. I checked and it's 4 spaces (automatically made by the tab). I will change that in vscode settings.